core/templates/services/svg-sanitizer.service.ts

```diff
@@ -183,9 +183,14 @@ export class SvgSanitizerService {
         node.remove();
       } else {
         for (let i = 0; i < node.attributes.length; i++) {
-          let nodeAttrName: string = node.attributes[i].name.toLowerCase();
+          const nodeAttrName: string = node.attributes[i].name;
+          const nodeAttrCombination = (
+            nodeTagName.toLocaleLowerCase() + ':' + nodeAttrName.toLowerCase());
```

Comment:
I think this could do with some detailed comments to explain what is going on and why the use of toLocaleLowerCase() is required. I don't see any explanation in either the code or the PR description.

In general, if there's logic which isn't likely to be clear to developers looking at the code, please err on the side of leaving an explanatory comment in the code. Thanks!